benchmarks/static_map/insert_bench.cu

```diff
@@ -54,10 +54,19 @@ std::enable_if_t<(sizeof(Key) == sizeof(Value)), void> static_map_insert(
 
   state.add_element_count(num_keys);
 
-  auto map = cuco::static_map{size, cuco::empty_key<Key>{-1}, cuco::empty_value<Value>{-1}};
-
-  state.exec([&](nvbench::launch& launch) {
+  state.exec(nvbench::exec_tag::timer, [&](nvbench::launch& launch, auto& timer) {
+    auto map = cuco::static_map{size,
+                                cuco::empty_key<Key>{-1},
+                                cuco::empty_value<Value>{-1},
+                                {},
+                                {},
+                                {},
+                                {},
+                                {},
+                                {launch.get_stream()}};
+    timer.start();
     map.insert_async(pairs.begin(), pairs.end(), {launch.get_stream()});
+    timer.stop();
```

Comment:
```suggestion
  state.exec(nvbench::exec_tag::timer, [&](nvbench::launch& launch, auto& timer) {
    timer.start();
    map.insert_async(pairs.begin(), pairs.end(), {launch.get_stream()});
    timer.stop();
    map.clear_async({launch.get_stream()});
```

This moves the memory allocation (which is also not stream-ordered, so you'd have to use `exec_tag::sync`; not ideal for accuracy) out of the hot loop.